src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2274,11 +2274,8 @@ protected override void BeginProcessing()
                 }
 
                 // write the ServiceController for the new service
-                using (ServiceController service =
-                    new ServiceController(Name)) // ensure dispose
-                {
-                    WriteObject(service);
-                }
+                ServiceController service = new ServiceController(Name); // ensure dispose
```

Comment:
It's probably worth removing the comment here since we're not disposing it anymore. 

A comment should probably be added explaining why we don't dispose it here.

Reply:
Done